ritasamir/assignment1/Q_01.java

```diff
@@ -0,0 +1,28 @@
+package assignment_1;
+
+import java.util.Arrays;
+
+public class Q_01 {
+
+	/**
+	 * checkAnagram is a method that takes two sentences and checks if they are
+	 * anagram or not 
+	 * @param firstStr is the first sentence
+	 * @param secondStr is the second sentence
+	 * @return true if they are anagram, false otherwise.
+	 */
+	public boolean checkAnagram(String firstStr, String secondStr) {
+	        if (stringSort(firstStr).equals(stringSort(secondStr))) {

```

Comment:
optional: Very nice! For the sake of future interviews, in case you get this questions, please have a read about other solutions (in particular answer number 4) [here](http://javaconceptoftheday.com/anagram-program-in-java/).